src/tests/HDFFile.cxx

```diff
@@ -66,6 +66,116 @@ class T_CommandHandler : public testing::Test {
     return event_index[i2] == cue_index;
   }
 
+  static void create_static_file_with_hdf_output_prefix() {
+    using namespace FileWriter;
+    using std::array;
+    using std::vector;
+    using std::string;
+    MainOpt &main_opt = *g_main_opt.load();
+    std::string hdf_output_prefix = "tmp-relative-output";
+    {
+#ifdef _MSC_VER
+      // NOTE just a suggestion, not tested on windows so far, we have no
+      // windows here.
+      _mkdir(hdf_output_prefix.c_str());
+#else
+      mkdir(hdf_output_prefix.c_str(), 0777);
+#endif
+    }
+    {
```

Comment:
How about refactoring this code block to a function which takes the main_opt reference and a string to use as the config_file contents? This would deduplicate some code and declutter a couple of your test cases.